.github/workflows/plugin-cd.yml

```
@@ -0,0 +1,72 @@
+on:
+  workflow_call:
+    inputs:
+      artifacts:
+        default: dist/*.tar.gz
+        description: |
+          The path or files of artifacts to upload
+        required: false
+        type: string
+
+      bucket:
+        default: mattermost-plugins-ci/release/
+        description: |
+          The S3 bucket full path to upload
+        required: false
+        type: string
+
+    secrets:
+      aws-access-key-id:
+        description: |
+          The AWS access key ID
+        required: true
+      aws-secret-access-key:
+        description: |
+          The AWS access secret key
+        required: true
+      github-token:
+        description: |
+          The Github token to publish artifacts
+        required: true
+
+permissions:
+  contents: read
+
+jobs:
+  release:
+    if: startsWith(github.ref, 'refs/tags')
```

```suggestion
    if: startsWith(github.ref, 'refs/tags/v')
```

Only release on tags starting with "v"